### PR TITLE
remove unused static images

### PR DESCRIPTION
**Pull request by jshaughn**

### CODE REVIEW

**Review by jmazzitelli — DISMISSED** [review]
scrub-a-dub-dub

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -2,54 +2,19 @@ name: CI
...
-          #   - Same reasoning as previous point.
-          URL_IGNORE="$URL_IGNORE,/^https:\/\/github.com\/kiali\/kiali\/pull\/\d+/"
-          URL_IGNORE="$URL_IGNORE,/^https:\/\/github.com\/kiali\/kiali\/issues\/\d+/"
-          URL_IGNORE="$URL_IGNORE,/^https:\/\/github.com\/kiali\/kiali\/tree\/v\d+\.\d+\//"
-          URL_IGNORE="$URL_IGNORE,/^https:\/\/github.com\/kiali\/kiali\/blob\/v\d+\.\d+\//"
-          URL_IGNORE="$URL_IGNORE,/.*web.libera.chat.*/"
-          htmlproofer --assume-extension --check-external-hash --empty_alt_ignore --url-ignore "$URL_IGNORE" ./public
-
+      - name: Prune
+        run: npm prune
```
Comment (jmazzitelli) [inline .github/workflows/ci.yaml]: Remove these two lines - they are no longer needed, but you need to first merge https://github.com/kiali/kiali.io/pull/462 and get that PR into yours. When that PR is in, you no longer need to do the prune outside of the container.

```suggestion
```

**Review by jmazzitelli — DISMISSED** [review]

**Comment by jshaughn** [comment]
@jmazzitelli There is one validation failure but I think it will go away when this gets merged, if not then I'll try and figure out the remaining issue.

**Review by jmazzitelli — APPROVED** [review]
lgtm

**Pull request merged**